mentors/models.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+from users.models import CustomUser
+
+
+class Mentor(models.Model):
+    name = models.CharField(max_length=256)
```

Comment:
@anshsrtv  make this uniuqe field so that same mentor is not added multiple times. Do similar changes in other models as well

Reply:
Add a created_at and modified_at field in all models